### PR TITLE
Updated the focus on the drag list and UI updates

### PR DESCRIPTION
**Pull request by natashapl**
### This pull request contains the following updates:

- Updated the draggable list focus to be the entire container rather than just the handle.
-  Updated the Read me.
- Attempted to fix the page number in the Pate title area but commented out my changes because I couldn't get them to past the storybook test. I need to consult with @danielnaab about this.

[PR Preview](https://federalist-16f2aca2-467c-449f-b725-5f1a0bd22dcd.sites.pages.cloud.gov/preview/gsa-tts/atj-platform/update-focus-on-drag-list-ui-updates/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ The platform is made up of the following high-level terms.
 
 ## Development
 
+This project uses the version of NodeJS that is in the .nvmrc. To ensure you're using the correct node version, run:
```
Comment (danielnaab) [inline README.md:33]:
```suggestion
This project uses the version of Node.js defined in [.nvmrc](./nvmrc). To ensure you're using the correct node version, you may use the [Node Version Manager (NVM)](https://github.com/nvm-sh/nvm):
```

Comment (danielnaab) [reply README.md:33]: I think a link to nvm would be helpful.

File: packages/design/src/FormManager/FormEdit/components/PreviewSequencePattern/DraggableList.tsx [hunk 2]
```diff
@@ -157,16 +155,19 @@ const SortableItem = ({
...
         transition,
-        opacity: isOver ? 0.4 : 1,
+        opacity: isOver ? 0.5 : 1,
         border: isOver ? '1px dashed #8168B3' : 'none',
+        outline: isOver ? 'none' : '',
       }}
     >
       <div className="grid-row draggable-list-item cursor-pointer">
         <div
           className="grid-col-12 width-full draggable-list-button cursor-grab padding-2"
```
Comment (danielnaab) [inline packages/design/src/FormManager/FormEdit/components/PreviewSequencePattern/DraggableList.tsx:165]: It seems `cursor-grab` maybe doesn't exist? The grabber has the pointer cursor.

**Review by ethangardner — APPROVED** [review]
@natashapl - tests seem to be working now and the items were addressed from Dan's previous comments. This looks good to me.

**Review by danielnaab — APPROVED** [review]

**Pull request merged**